### PR TITLE
upload function

### PR DESCRIPTION
**Pull request by ndittren**

### CODE REVIEW

File: mediathread/templates/assetmgr/install_safari_extension.html [hunk 1]
```diff
@@ -1,26 +1,11 @@
 {% load staticfiles %}
 
-<div id="safari-install-container">
-    <h4><strong>Install Safari extension</strong></h4>
+<div id="firefox-install-container">
```
Comment (sdreher) [inline mediathread/templates/assetmgr/install_safari_extension.html:3]: This page is for the Safari extension, not Firefox. I think we wanted to update both Firefox & Safari pages to direct them to Chrome.

File: mediathread/templates/main/collection_add.html [hunk 2]
```diff
@@ -39,14 +41,57 @@ <h2>Upload Media</h2>
...
                         </p>
-                        <h3>Video</h3>
-
-
-
-                        <h3>Audio</h3>
-
-
 
+                        {% if can_upload and uploader %}
```
Comment (sdreher) [inline mediathread/templates/main/collection_add.html:45]: The layout for the Upload section is a bit wonky...I think there are just some mismatched divs.

File: mediathread/templates/main/collection_add.html [hunk 3]
```diff
@@ -33,40 +35,89 @@
...
+                                                        <div class="col-auto my-1">
+                                                            <label class="mr-sm-2 sr-only" for="video_owners"></label>
+                                                            <select class="custom-select mr-sm-2" id="video_owners" name="as">
+                                                            {% for owner in owners %}
+                                                                <option value="{{owner.username}}" {% ifequal owner.username user.username %} selected="selected"{% endifequal %}>{{owner.public_name}}</option>
+                                                            {% endfor %}
+                                                            </select>
+                                                        </div>
+                                                    {% endif %}
+                                                    <div classs="col-auto my-1">
```
Comment (sdreher) [inline mediathread/templates/main/collection_add.html:73]: `class` typo here.

File: mediathread/templates/main/collection_add.html [hunk 4]
```diff
@@ -33,40 +35,89 @@
...
+                                                        <button type="submit" class="btn btn-secondary upload_button video">Upload Video</button>
+                                                    </div>
+                                                </div>
+                                        </form>
+                                        <h3>Audio</h3>
```
Comment (sdreher) [inline mediathread/templates/main/collection_add.html:78]: Add a little spacing here - maybe `mt-1 or 2`

File: mediathread/templates/main/collection_add.html (hunk 3 above)
Comment (ndittren) [reply mediathread/templates/main/collection_add.html]: Ah!

File: mediathread/templates/main/collection_add.html (hunk 4 above)
Comment (ndittren) [reply mediathread/templates/main/collection_add.html]: Got it!

**Pull request merged**